ReactSkia/components/RSkComponentActivityIndicator.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#include "include/core/SkPaint.h"
+#include "include/core/SkPath.h"
+
+#include <ReactSkia/activityindicator/react/renderer/components/activityindicator/ActivityIndicatorProps.h>
+#include "ReactSkia/components/RSkComponentActivityIndicator.h"
+#include "ReactSkia/views/common/RSkConversion.h"
+
+#include "ReactSkia/utils/RnsLog.h"
+
+#define ACTIVITY_INDICATOR_DEFAULT_ARC_COLOR                      0xFF999999 /*As per IOS documentation*/                       
```

Comment:
Can we set as SKCOLOR    SkColorSetARGB(0xFF, 0x99, 0x99, 0x99) //Dark Gray, which will be more meaningful